src/master/util.ts

```diff
@@ -187,10 +188,12 @@ export async function loadChannel(
                             .reverse();
                         messages = messages.slice(messages.length - options.maxMessagesPerChannel);
                         for (const msg of messages) {
-                            webhook.send(msg.content, {
+                            let sentMsg: Message = await webhook.send(msg.content, {
```

Comment:
Here, I think `const` is more approriate.
```suggestion
                            const sentMsg: Message = await webhook.send(msg.content, {
```

Reply:
Also, I think specify the message type is not needed, no?

Reply:
Personally, I always specify the type when declaring variables, but since it doesn't fit with your code style, I have changed it 👍